.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,13 @@
+- [ ] I ensure that this feature or anything similar to this does not exist in the website. 
+
+
+**Does your feature request solve any on going issue? Please describe.**
+
+**Describe the uses of this feature**
+
+A clear and concise description of what you want to happen.
+
+
+**Additional context**
+
+Add any other context or screenshots about the feature request here.
```

Comment:
convert line 1, 8 and 13 to comment

Reply:
Line 1 is a checkbox, do you want me to erase it or comment it?